pkg/tbtc/chain.go

```diff
@@ -26,6 +29,15 @@ type DistributedKeyGenerationChain interface {
 	OnDKGStarted(
 		func(event *DKGStartedEvent),
 	) subscription.EventSubscription
+	OnDKGResultSubmitted(
+		func(event *DKGResultSubmittedEvent),
+	) subscription.EventSubscription
+	SubmitResult(
+		result *dkg.Result,
+		signatures map[group.MemberIndex][]byte,
+		startBlockNumber uint64,
+		participantIndex group.MemberIndex,
+	) error
```

Comment:
Let's separate functions with blank lines and document them.

Reply:
Done in 1c60971644aeb94e2222261931a1d94f2d265d03.